livekit-agents/livekit/agents/utils/misc.py

```diff
@@ -43,3 +44,7 @@ def merge_frames(buffer: AudioBuffer) -> rtc.AudioFrame:
 
 def time_ms() -> int:
     return int(time.time() * 1000)
+
+
+def nanoid() -> str:
```

Comment:
as a public API, it's not clear what `nanoid` is.. are you looking for a `shortuuid` ?